CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+### [3.22.3] 2022-10-04
+
+### Changed
+
+- Fix: Replace % character by %% on `pandas_read_sql` to prevent "dict is not a
+  sequence" error when passing a matches/not-matches query.
```

Comment:
```suggestion
- Fix: Replace % character by %% in `pandas_read_sql` to prevent pandas from interpreting `%` as the interpolation
  of an SQL parameter
```
This would have been problematic in other cases than for translated match queries

Reply:
[`35c92f2` (#708)](https://github.com/ToucanToco/toucan-connectors/pull/708/commits/35c92f2140f63f1eb2481219a36ba21079a77c5e)